### PR TITLE
[minor] DDF-04752 Update PULL_REQUEST_TEMPLATE - remove "Any background context you want to provide?"

### PR DESCRIPTION
**Pull request by rymach**
#### Who is reviewing it? 

@ryeats 
@troymohl 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@AzGoalie 

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4752 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
imo we can get rid of the checklist as well.  it's not really used.  @Bdthomson @Schachte @mojogitoverhere @jrnorth

**Comment by rymach** [comment]
> imo we can get rid of the checklist as well. it's not really used. @Bdthomson @Schachte @mojogitoverhere @jrnorth

less is more! :D

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -86,11 +85,6 @@ Fixes: #____
 
 #### Screenshots
 <!--(if appropriate)-->
-#### Checklist:
```
Comment (ethantmanns) [inline .github/PULL_REQUEST_TEMPLATE]: ❗️ I disagree about removing this. I always check this when reviewing PRs. Does everyone else feel like they're not used?

**Comment by rzwiefel** [comment]
I mean why don't we just delete everything and let people put what they need each time?

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (peterhuffer) [reply .github/PULL_REQUEST_TEMPLATE]: I personally like the documentation one and the test ones. If at the very least they make the developer consider if they forgot any associated docs/tests updates that should happen in addition to the code changes.

Comment (peterhuffer) [reply .github/PULL_REQUEST_TEMPLATE]: I like this section. If reviewers don't use them, at the very least it forces the developer to consider if any areas mentioned in the checklist should be updated. Every item in this check list should be considered when making code changes.

**Comment by rzwiefel** [comment]
Okay that was more of a thought question than actual suggestion. A template is meant to be things so you don't have to do it over and over. 
image

People can `cmd+a` (or `ctrl+a`) and `backspace` if they want a blank template.

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -76,7 +76,6 @@ If you don't know who to ask, you can request reviews in https://groups.google.c
 
 #### How should this be tested?
 <!--(List steps with links to updated documentation)-->
-#### Any background context you want to provide?
```
Comment (coyotesqrl) [inline .github/PULL_REQUEST_TEMPLATE]: ❓ Why? Do we often commit changes to our code base without context? Do we change our software based on a random number generator or perhaps a Magic 8-Ball :tm:?

Snark aside, I do not fundamentally understand the reasoning behind this change. Looking at the Issue for it, it seems that perhaps it too was generated by a Magic 8-Ball :tm: that hadn't yet been programmed to fill out the Issue. Huh. I guess snark isn't completely put aside after all.

**Comment by rzwiefel** [comment]
Also the point of most of these things is to make people think about them if they havent -- which they wont do if they are missing.

**Comment by Lambeaux** [comment]
I know it's already closed but I'm throwing in my two cents anyway. I think background context is actually something we _struggle_ with when it comes to explaining PRs. I'd consider adding questions to the template before removing it; to help facilitate more precise details and testing procedures. 

Reviewing code is trivial when compared to reviewing solutions. And I can't review a solution without the background context and how the user triggers the functionality.

**Pull request closed without merging**